gnc/sub8_trajectory_generator/include/sub8_trajectory_generator/sub8_state_space.h

```diff
@@ -7,7 +7,10 @@
 
 #include "ompl/base/StateSpace.h"
 #include "ompl/base/spaces/RealVectorStateSpace.h"
+#include "tgen_common.h"
+#include <Eigen/Dense>
 
+using namespace Eigen;
 using ompl::base::CompoundStateSpace;
```

Comment:
Avoid doing  
    
    using namespace 

[In a header](http://stackoverflow.com/questions/167862/how-can-i-unuse-a-namespace)